### ISSUE TITLE
Create Event Card

### ISSUE THREAD

**Issue by HassanBahati** [osckampala/evently#22]
we need to add to create a card component and add to the home page.

here is a reference https://v1.tailwindcss.com/components/cards#stacked
![Screenshot from 2023-03-24 11-49-19](https://user-images.githubusercontent.com/65954740/227470399-1e6aac59-559e-4c0f-8f0b-d428f90d889d.png)

the card should have the following 

- image
- event tags (eg #flutter, #cloud, etc)
- event title
- event description

Best if we can create a sample data structure that contains sample events objects and loop those events through the card....so that multiple cards appear on the home page

**Comment by mosespace** [comment]
Hi, @HassanBahati But I think you did this in the `page.tsx` Is it it still open? or now we should do it in a data structure format?

**Comment by HassanBahati** [comment]
hi @mosespace yes partly...though that task also involves reading the data from a data structure that should reside in [this directory](https://github.com/OSCA-Kampala-Chapter/evently/tree/main/content)

**Comment by mosespace** [comment]
@HassanBahati let me have a look at this,

**Comment by HassanBahati** [comment]
alright

**Comment by HassanBahati** [comment]
@mosespace how far on this?

**Comment by mosespace** [comment]
@HassanBahati hopefully am handling this tonight! 🙏

**Comment by mosespace** [comment]
Hi, @HassanBahati , Unfortunately I might be getting errors on this! any documentations / guidance on this?

**Comment by HassanBahati** [comment]
> Hi, @HassanBahati , Unfortunately I might be getting errors on this! any documentations / guidance on this?

Hi @mosespace sorry about that...what errors do you get? 
1. Does importing the data in `events.json` into `index.tsx` work?

**Comment by mosespace** [comment]
@HassanBahati , yes it works, but when i try replace the hard-coded event data in the Card component with the imported data, like using this brackets `{}` for example `{events.name}` it fails! i also do the mapping! but it still fails

**Comment by HassanBahati** [comment]
> @HassanBahati , yes it works, but when i try replace the hard-coded event data in the Card component with the imported data, like using this brackets `{}` for example `{events.name}` it fails! i also do the mapping! but it still fails

@mosespace please open a pull request with what you have so we can see what the issue could be
